Controller/ConnectController.php

```diff
@@ -359,4 +348,60 @@ protected function getTemplatingEngine()
     {
         return $this->container->getParameter('hwi_oauth.templating.engine');
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Symfony <2.6 BC. To be removed.
+     */
+    protected function isGranted($attributes, $object = null)
+    {
+        if (method_exists('Symfony\Bundle\FrameworkBundle\Controller\Controller', 'isGranted')) {
```

Comment:
I use this method cause in any case I would have to switch services to avoid deprecated one (security.context).

In this case, I would prefer to let Symfony controller do the stuff.